.github/workflows/rm-closed-pr-images.yaml

```diff
@@ -0,0 +1,28 @@
+name: Remove docker images for stale/closed PR(s).
+on:
+    pull_request:
+        branches:
+            - main
+        types: [closed]
+jobs:
+    cleanup:
+        name: PR - cleanup pr-${{ github.event.number }} images
+        runs-on: ubuntu-latest
+        permissions:
+            packages: write
+        strategy:
+            matrix:
+                image:
+                    - anvil
+                    - machine-emulator-tools
+                    - rollups-node
+                    - sdk
+        steps:
+            - uses: vlaurin/action-ghcr-prune@v0.5.0
+              with:
+                  organization: sunodo
+                  container: ${{ matrix.image }}:pr-${{ github.event.number }}
```

Comment:
don't you need a `ghcr.io/${{ gitHub.repository_owner }}` here?

Reply:
I don't think so.